sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -381,4 +381,85 @@ public void decommissionModel(String modelId) {
     public Response<Void> decommissionModelWithResponse(String modelId, Context context) {
         return digitalTwinsAsyncClient.decommissionModelWithResponse(modelId, context).block();
     }
+
+    //==================================================================================================================================================
+    // Component APIs
+    //==================================================================================================================================================
+
+    /**
+     * Get a component of a digital twin.
+     * @param digitalTwinId The Id of the digital twin to get the component from.
+     * @param componentPath The path of the component on the digital twin to retrieve.
+     * @return The application/json string representing the component of the digital twin.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public String getComponent(String digitalTwinId, String componentPath) {
+        return digitalTwinsAsyncClient.getComponent(digitalTwinId, componentPath).block();
```

Comment:
We've been following the format where the async APIs call the async overloads, and the max overload API calls the PL.
Similarly, the sync APIs call the sync overloads, and the max overload sync API calls  the async API.

Reply:
Gotcha, I can chain it that way instead

Reply:
Fiyi - for default context you can use `Context.NONE`.